components/Hero/StoryHeroStacked.styles.ts

```diff
@@ -2,7 +2,7 @@ import { cnb } from 'cnbuilder';
 
 export const root = 'relative';
 
-export const contentWrapper = (hasVideo: boolean) => cnb('mt-40 md:-mt-60 xl:mt-0', hasVideo && 'rs-pb-3');
+export const contentWrapper = (hasVideo: boolean, hasMedia: boolean) => cnb('mt-40 md:-mt-60 xl:mt-0', hasVideo && 'rs-pb-3', !hasMedia && 'rs-pb-9');
```

Comment:
Small adjustment to the "stacked" variant of the hero. If no image/video is added (this sits below the hero content - h1, byline, intro etc), add a large bottom padding

Reply:
Nitpick: hasVideo and hasMedia is slightly confusing as it isn't immediately clear what the difference between the two are. Perhaps a code comment to go along with this would be helpful.